Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:latest AS build
+
+WORKDIR /code
+
+COPY package.json ./
+
+RUN npm install
+
+COPY . .
+
+ENTRYPOINT [ "npm", "run", "start" ]
```

Comment:
You'll need to export port 4000 and manually copy the `.env` file / add a way to expose environment variables to the container, like so:
```dockerfile

COPY . .

COPY .env ./

ENTRYPOINT [ "npm", "run", "start" ]

EXPOSE 4000
```

Reply:
Sure,

Reply:
I add almost a commentaries to better explanation